frontend/ts/ChitListExtension.ts

```diff
@@ -0,0 +1,25 @@
+import naja from "naja";
+
+export function initializeSendMassForm(container: Element, checkboxPrefix: string): void {
+    container.querySelectorAll<HTMLFormElement>('.formMass').forEach(form => {
+        form.addEventListener('submit', (event:any) => {
+            const formData = new FormData(form);
```

Comment:
Nestálo by za to to odesílání formuláře vyextrahovat do samotné funkce pro vyšší přehlednost?


Že by tady zůstalo něco jako:

```ts
export function initializeSendMassForm(container: Element, checkboxPrefix: string): void {
    container.querySelectorAll<HTMLFormElement>('.formMass').forEach(form => {
        form.addEventListener('submit', (event:any) => {
            submitMassAddEvent(form); // <- Tady by bylo to vytváření formdata a odeslání

            event.preventDefault();
        }
    }
}
```

Reply:
upraveno